app/user.go

```diff
@@ -1066,6 +1066,34 @@ func (a *App) UpdateUser(user *model.User, sendNotifications bool) (*model.User,
 	return rusers[0], nil
 }
 
+func (a *App) UpdateUserActive(userId string, active bool) (bool, *model.AppError) {
```

Comment:
I think this methods overcomplicates the problem. You can just do
```go
user,err := a.GetUser(userId)
if err != nil{
	return err
}

if _, err = UpdateActive(user, active) {
  return err
}
return nil
```
in `(api *PluginAPI) UpdateUserActive()`

Reply:
Was not sure if I had to do all the checks done on the actual user.go implemementation. Changed the code accrodingly. This looks much cleaner now.